### ISSUE TITLE
(OSX) "There was an error trying to install the mountdir." AND "There was an error trying to install the redirector"

### ISSUE THREAD

**Issue by moo-im-a-cow** [keybase/client#11240]
Hi
After every reboot, i get 2 errors in a single pop up.
![ScreenShot of error](https://user-images.githubusercontent.com/5210502/38348687-9cc51f48-38e6-11e8-8a6f-a90ac1f983a2.png)

I have not tried re installing keybase yet (wanted to get the devs a log to prevent this in the future)

LOG ID: (not here yet)

I will try re installing keybase after posting this, and i will update if it works after that.

**Comment by moo-im-a-cow** [comment]
Ran `$ keybase uninstall` then `$ rm -rf /Applications/keybase.app`
then i downloaded and installed keybase, havent had the error since.

**Comment by strib** [comment]
Hmm, thanks.  Could be related to #11197.  @moo-im-a-cow after you re-installed, did you also re-add the file system drivers by clicking the "Display in Finder" link from the Folders tab?

**Comment by moo-im-a-cow** [comment]
yes i did click "Display in Finder".

just realised that the icon is different now, and the name on my desktop has changed from "Keybase" to "Keybase (username)" where "username" is my computer username (not keybase account name)

**Comment by strib** [comment]
Yes, the move is intentional, but `/keybase` should still be working.

If everything seems to be working fine now, feel free to close this out.
